### DISCUSSION TITLE
plc2

### DISCUSSION THREAD

**Discussion by Chormore-EDGE** [Meloong-Git/PCL#1566] [讨论]
![image](https://user-images.githubusercontent.com/114737601/229487088-25be7af5-94ff-4d46-9997-7448310d4c62.png)

**Comment by WForst-Breeze** [comment]
~平流层 II 启动器~ 🤔🤔🤔

**Comment by iPodtouch6557** [comment]
可不可以对plc2用户给予欧皇彩？
只是个想法所以不交issue
~不对我可以在个性化-标题栏里我命由我不由天~

**Reply by WForst-Breeze** [reply]
应该不会，这个更多算是彩蛋性质的 ~尽管的确能刷到 PLC2 的都挺欧的~
